packages/components/src/components/dropdown/utility.ts

```diff
@@ -0,0 +1,45 @@
+export type TListItem = {
+    tooltip?: string;
+    disabled?: boolean;
+    has_tooltip?: boolean;
+    text: string;
+    value: string;
+};
+
+export type TList = TListItem[];
+
+type TActiveNode = {
+    attributes: {
+        tabIndex: number;
+    };
+    nextSibling: TActiveNode;
+    previousSibling: TActiveNode;
+};
+
+type TFocusableNode = (active_node: TActiveNode) => any;
```

Comment:
```suggestion
type TFocusableNode = (active_node: TActiveNode) => (TActiveNode | null);
```
@niloofar-deriv could you please try this?

Reply:
I tried that before but it didn't work

Reply:
@niloofar-deriv this will work if you add `HTMLElement &` to line 11🙂 https://github.com/niloofar-deriv/deriv-app/pull/52/files#r1097034197

Reply:
Thank u so much Mariya, now it is working :partying_face: